services/microcredit/src/borrowers.ts

```diff
@@ -63,3 +66,62 @@ export async function updateBorrowers(): Promise<void> {
         utils.slack.sendSlackMessage('🚨 Error to update borrowers', config.slack.lambdaChannel);
     }
 }
+
+export async function updateCurrentDebt(): Promise<void> {
+    utils.Logger.info('Updating current debt...');
+    const { subgraphMicroCreditBorrowers, appUser } = database.models;
+    const t = await database.sequelize.transaction();
+
+    try {
+        // get borrowers that did not repay for more that 3 weeks
+        const threeWeeksAgo = new Date();
+        threeWeeksAgo.setDate(threeWeeksAgo.getDate() - 21);
+
+        const borrowers = await subgraphMicroCreditBorrowers.findAll({
+            include: [{
+                model: appUser,
+                as: 'user',
+            }],
+            where: {
+                lastRepayment: {
+                    [Op.lt]: threeWeeksAgo.getTime() / 1000 | 0
```

Comment:
Just thought of this now. It will start to update every day then, but it shouldn't. Do we have an "updatedAt" on this table. If so, can we update the debt, only if the lastRepayment and updatedAt are both older than 3 weeks?

Reply:
Got it!
I changed to check the updatedAt and lastRepayment